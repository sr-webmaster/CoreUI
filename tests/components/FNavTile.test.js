import { mount } from '@vue/test-utils'
import Component from '@freshinup/core-ui/src/components/FNavTitle.vue'

describe('FNavTile', () => {
  it('snapshot default', async () => {
    const wrapper = mount(Component)
    expect(wrapper.element).toMatchSnapshot()
  })
  it('snapshot with color', async () => {
    const wrapper = mount(Component, {
      propsData: {
        color: 'info',
        value: true
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
