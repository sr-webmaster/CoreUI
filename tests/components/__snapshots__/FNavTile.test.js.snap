// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FNavTile snapshot default 1`] = `
<div
  role="listitem"
>
  <div
    class="v-list__tile theme--light"
  />
</div>
`;

exports[`FNavTile snapshot with color 1`] = `
<div
  role="listitem"
  value="true"
>
  <div
    class="v-list__tile v-list__tile--active theme--light info infotext--text"
  />
</div>
`;
